addons/featured-dangos-v2/userscript.js

```diff
@@ -0,0 +1,47 @@
+export default async function ({ addon, console, msg }) {
+  // Fetch projects from a studio
+  const fetchStudio = async (studio) => {
+    const response = await fetch(`https://api.scratch.mit.edu/studios/${studio}/projects/`);
+    return response.json();
+  };
+
+  const projects = await fetchStudio("33286407");
+
+  // Wait until Featured Projects row loads
+  const firstThumbnail = await addon.tab.waitForElement(".carousel .thumbnail");
+
+  // Get Featured Projects row <div>
+  const featuredProjectsRow = firstThumbnail.closest(".box");
+
+  // Clone Featured Projects row and its children
+  const featuredDangosRow = featuredProjectsRow.cloneNode(true);
+  featuredDangosRow.classList.add("sa-featured-dangos-row");
+
+  // At this point, the Featured Dangos row might be 5 "Project" placeholders linking to
+  // /projects/1, or possibly the actual featured projects. We will be handling both cases.
+
+  // Change row title
+  featuredDangosRow.querySelector(".box-header > h4").textContent = msg("row-title");
```

Comment:
The addon ID also needs to be changed in the file name of `addons-l10n/en/featured-dangos.json`, or this won't properly get the message.